src/app/about/about.component.html

```diff
@@ -55,21 +55,21 @@ <h2 class="bold">About Us</h2>
           <h5 class="bold">About YaCy</h5>
           <p>YaCy is a free search engine that anyone can use to build a search portal for their intranet or to help search
             the public internet. Read more about YaCy here -
-            <a class="page-link" href="https://yacy.net/en/index.html" target="_blank">YaCy Decentralized Web Search</a>
+            <a class="page-link" href="https://{{yacyUrl}}/en/index.html" target="_blank">YaCy Decentralized Web Search</a>
           </p>
         </div>
 
         <div class="col-lg-4 col-sm-12 sub-details">
           <h5 class="bold">Communication</h5>
           <p>Facing issues while setting up project on local environment? Our chat channel is on gitter here:
-            <a class="page-link" href="https://gitter.im/fossasia/susper.com" target="_blank">fossasia/susper.com</a>. We'll be happy to help you out!'</p>
+            <a class="page-link" href="https://gitter.im/fossasia/{{susperUrl}}" target="_blank">fossasia/susper.com</a>. We'll be happy to help you out!'</p>
         </div>
 
         <div class="col-lg-4 col-sm-12 sub-details">
           <h5 class="bold">Contribute to our project</h5>
           <p>Get involved as an Open Source developer, designer or tester and start your adventure today! Solve an issue or
             feature request on our repositories with
-            <a class="page-link" href="https://github.com/fossasia/susper.com" target="_blank">FOSSASIA</a>. Build up your developer profile and become part of a fantastic community.</p>
+            <a class="page-link" href="https://github.com/fossasia/{{susperUrl}}" target="_blank">FOSSASIA</a>. Build up your developer profile and become part of a fantastic community.</p>
```

Comment:
If I search ```https://github.com/fossasia```, then also I would get many of its instance. Could you add it too ?